charts/controller/templates/controller-deployment.yaml

```diff
@@ -125,6 +125,10 @@ spec:
                   key: password
             - name: RESERVED_NAMES
               value: "deis, deis-builder, deis-workflow-manager, grafana"
+            - name: WORKFLOW_NAMESPACE
```

Comment:
It's k8s magic through the downward API--I just followed the example here: https://kubernetes.io/docs/user-guide/downward-api/

Kubernetes will ensure that `$WORKFLOW_NAMESPACE` is populated by the deployment's actual namespace when deployed since we reference `metadata.namespace`.